src/patternfly/components/Truncate/examples/Truncate.md

```diff
@@ -8,44 +8,83 @@ import './Truncate.css'
-    {{> truncate-end truncate-end--text='sit amet molestie est accumsan.'}}
+    {{> truncate-start truncate-start--text='redhat_logo_black_and_white_reversed_simple_'}}
+    {{> truncate-end truncate-end--text='with_fedora_container.zip'}}
   {{/truncate}}
 </div>
 ```
 
 ### Start
+
+When only the `.pf-v6-c-truncate__end` element is used, truncation will occur at the start of the string. `&lrm;` **must** be included at the end of a string to denote the ending punctuation mark, otherwise it will render at the start of the truncated content.
+
 ```hbs
 <div class="pf-v6-c-truncate--example">
   {{#> truncate truncate--id='start-truncation-example'}}
-    {{> truncate-end truncate-end--text='Vestibulum interdum risus et enim faucibus, sit amet molestie est accumsan.&lrm;'}}
+    {{> truncate-end truncate-end--text='redhat_logo_black_and_white_reversed_simple_with_fedora_container.zip'}}
   {{/truncate}}
 </div>
 ```
 
+### Based on max characters
+
+Apply the `pf-m-ignore-resizing` class to the `pf-v6-c-truncate` element to implement truncation based on an amount of characters rather than a parent container width. You must ensure that the omission designator (typically an ellipsis) is removed from the accessibility tree by wrapping it in a `<span>` with the `aria-hidden="true"` attribute.
+
+```hbs
+<div>Truncated at end position:</div>
+{{#> truncate truncate--ShouldIgnoreResize=true}}
+  <span>redhat_logo_black_and_white_reversed_simple_with_</span>
+  <span aria-hidden="true">…</span>
+  {{#> screen-reader}}fedora_container.zip{{/screen-reader}}
+{{/truncate}}
+<br />
+<br />
+<div>Truncated at middle position:</div>
+{{#> truncate truncate--ShouldIgnoreResize=true}}
+  <span>redhat_logo_black_and_</span>
+  {{#> screen-reader}}white_reversed_simple_with_{{/screen-reader}}
+  <span aria-hidden="true">…</span>
+  <span>fedora_container.zip</span>
+{{/truncate}}
+<br />
+<br />
+<div>Truncated at start position:</div>
+{{#> truncate truncate--ShouldIgnoreResize=true}}
+  {{#> screen-reader}}redhat_logo_black_{{/screen-reader}}
+  <span aria-hidden="true">…</span>
+  <span>and_white_reversed_simple_with_fedora_container.zip</span>
+{{/truncate}}
+```
 
 ## Documentation
 
 ### Usage
 
 | Class | Applied | Outcome |
 | -- | -- | -- |
-| `.pf-v6-c-truncate` | `<span>` | Initiates the truncate component. |
+| `.pf-v6-c-truncate` | `<span>` | Initiates the truncate component. **Required** |
 | `.pf-v6-c-truncate__start` | `<span>` | Defines the truncate component starting text. |
 | `.pf-v6-c-truncate__end` | `<span>` | Defines the truncate component ending text. |
+| `.pf-m-ignore-resizing` | `span.pf-v6-c-truncate` | Modifies the truncate component to ignore resizing and container widths for truncation. |
+| `.pf-v6-screen-reader` | `.pf-v6-c-truncate span` | Visually hides text when truncation is meant to occur based on a maximum amount of characters rather than resizing/container width. | 
```

Comment:
This docs table typically only lists classes for the component. We would have traditionally put a line like this in the a11y docs table.

Reply:
Just to confirm, is this in regards to the screen reader class row, or both the added rows?

Reply:
Sorry about that - just the screen reader class row. Though on `.pf-m-ignore-resizing`, is there a reason you added `span.pf-v6-c-truncate` and not just `.pf-v6-c-truncate`?

And I'm not wild about `.pf-m-ignore-resizing`, but I'm not wild about naming things either, so I didn't mention it 😂 WDYT about `.pf-m-static` or `.pf-m-fixed` (as in fixed width/characters) instead?

Reply:
1. I may have been thinking of the screen reader one in reverse 😆 no real reason to have it written that way, though, you're right
2. Both of those sound much better (though the next hill is to decide on which one). Do you think we need to worry about the lack of context behind them at all? Like `pf-m-fixed` may invoke the thought of "fixed dimensions" or 'fixed position', but the context is like you said "fixed characters" more than anything. The docs would of course relay the context, was just curious.